Makefile

```diff
@@ -19,6 +19,21 @@ build: deps
 		&& export CGO_ENABLED=0 \
 		&& go build -v -mod=vendor -ldflags $(BUILD_FLAGS) -o ${BINARY} ./cli/main.go
 
+install: build
+	@echo "=> Preparing directories and configs"
+	@id neo-go || useradd -s /usr/sbin/nologin -d /var/lib/neo-go neo-go \
```

Comment:
Please leave this setup to RPM/DEB postinstall scripts (hmm, we're missing issues for these now).

Reply:
done